pycontrails/core/met.py

```diff
@@ -1022,6 +1037,84 @@ def to_vector(self, transfer_attrs: bool = True) -> vector_module.GeoVectorDatas
             vector.attrs.update({str(k): v for k, v in self.attrs.items()})
         return vector
 
+    def _get_attr_template(
+        self,
+        name: str,
+        supported: tuple[str, ...],
+        examples: dict[str, str],
+    ) -> str:
+        """Look up an attribute with a custom error message."""
+        try:
+            out = self.attrs[name].upper()
+        except KeyError as e:
+            msg = f"Specify '{name}' attribute on underlying dataset."
+
+            for i, (k, v) in enumerate(examples.items()):
+                if i == 0:
+                    msg = f"{msg} For example, set attrs['{name}'] = '{k}' for {v}."
+                else:
+                    msg = f"{msg} Set attrs['{name}'] = '{k}' for {v}."
+            raise KeyError(msg) from e
+
+        if out not in supported:
```

Comment:
I would make `supported` all `.upper()` to make it truly case insensitive

Reply:
c95b2b5b2ecfe111193fc8128c7a3b6386f0d880